src/shared/components/layouts/UniProtHeader.tsx

```diff
@@ -141,7 +141,7 @@ const UniProtHeader = () => {
     <Header
       links={displayedLinks}
       isNegative={isHomePage}
-      search={shouldShowSearch && <SearchContainer />}
+      search={shouldShowSearch ? <SearchContainer /> : undefined}
```

Comment:
You're right, but it's just because the Franklin component's PropTypes would throw a warning if it gets a `false` value:
```
Warning: Failed prop type: Invalid prop `search` of type `boolean` supplied to `Header`, expected a single ReactElement.
```